src/OpenTracing/NoopTracer.php

```diff
@@ -15,7 +15,7 @@ public static function create()
 
     public function startSpan(
         $operationName,
-        SpanReference $parentReference = null,
+        Reference $parentReference = null,
```

Comment:
I guess this is going to be removed because of https://github.com/opentracing/opentracing-php/pull/15/files#r132735269, WDYT @felixfbecker?